charts/mattermost-rtcd/values.yaml

```diff
@@ -95,4 +97,15 @@ tolerations: {}
 
 dnsConfig: {}
 
-affinity: {}
+affinity:
```

Comment:
@d-wierdsma We need an affinity here that ensures that not both pods will be scheduled in the same rtcd node as calls will not work with that.

Reply:
Updated change